sdk/metricsadvisor/Azure.AI.MetricsAdvisor/CHANGELOG.md

```diff
@@ -2,14 +2,24 @@
 
 ## 1.0.0-beta.5 (Unreleased)
 
+### New Features
+- `DimensionKey` now implements the `IEnumerable<KeyValuePair<string, string>>` interface. Dimension columns can now be enumerated.
+- Added method `Contains` to `DimensionKey` to check whether or not a dimension column is present.
```

Comment:
Do you think it is worth showing this in any sample? or the use of the `TryGetValue`? The names and docstrings are clear enough that I don't think it is needed, but asking just in case

Reply:
To tell you the truth, I don't think these methods will be used that much. In general, customers will need to go through every element anyway, so they'll use the enumerator (kudos to KC who suggested making it an IEnumerable). I added these methods just in case someone wants to check a specific element without going through the whole set.